.github/workflows/daily-lighthouse-scan.yml

```diff
@@ -28,7 +28,7 @@ jobs:
           # To omit urls from being scanned in lighthouse, add rootUrl inside IN() function
           REGISTRY_ARRAY=$(jq '[.[]
                       | select((.rootUrl
-                                | IN("/health-care/health-questionnaires/questionnaires/answer-questions", "/health-care/health-questionnaires/questionnaires")
```

Comment:
I am not sure about this line. Can/ should the `IN` be removed?

Reply:
the command can be changed to

```
jq '[.[]
                      | select((.rootUrl) and .template.vagovprod != false)
                      | .rootUrl ]' ./content-build/src/applications/registry.json
```

and get the same output. (once hcq is gone)

It does also work with empty quotes in the `IN` statement as you have it.